site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -60,12 +60,19 @@ kubectl config use-context manager-cluster
 ```
 {{% /alert %}}
 
-### Jobset CRD only install
+### JobSet installation
 
-As mentioned in the [MultiKueue Overview](/docs/concepts/multikueue/#jobset) section, in the manager cluster only the JobSet CRDs should be installed, you can do this by running:
+If you are using Kueue in version 0.7.0 or newer install the JobSet on the
+management cluster (see [JobSet Installation](https://jobset.sigs.k8s.io/docs/installation/)
+for more details). We recommend using JobSet 0.5.1+ for MultiKueue.
```

Comment:
Version 0.5.0 is needed , it should not be just a recommendation.

Reply:
Yeah, I understand that the support started from 0.5.0. but it contains some relevant bugs:
- https://github.com/kubernetes-sigs/jobset/pull/528
- https://github.com/kubernetes-sigs/jobset/pull/527

Actually, our users already hit the issues, which made the investigations unnecessarily harder. Thus we bumped the version suggested in the JobSet page: https://github.com/kubernetes-sigs/jobset/pull/577.

I would prefer not to mention the old version at all in the docs, because this may increase a change for some users installing 0.5.0.

Reply:
My problem was with "We recommend to ... ", multikueue is not working with versions lower then 0.5 .